api/Makefile

```diff
@@ -84,3 +84,17 @@ django-collect-static:
 .PHONY: serve
 serve:
 	poetry run gunicorn --bind 0.0.0.0:8000 app.wsgi --reload
+
+.PHONY: generate-ld-client-types
+generate-ld-client-types:
+	curl -sSL https://app.launchdarkly.com/api/v2/openapi.json | \
+	npx openapi-format /dev/fd/0 \
+		--filterFile ld-openapi-filter.yaml | \
+	datamodel-codegen \
+		--output integrations/launch_darkly/types.py \
+		--output-model-type typing.TypedDict \
+		--target-python-version 3.10 \
```

Comment:
This should be 3.11, right? Because that's what we use in production? https://github.com/Flagsmith/flagsmith/blob/2c56526706ea48b48714f03103bbb4de1db7ba3a/api/Dockerfile#L3C3-L3C3

Reply:
Enterprise image uses 3.10, so it's the lowest common denominator here.

3.10 uses `typing_extensions` import for `NotRequired` annotations, so tests actually fail on 3.10 if we use 3.11 argument here.